README.md

```diff
@@ -2,152 +2,46 @@
+Refer to [PlotCreating](docs/PlotCreating.md) for the complete documentation.
 
-git clone https://github.com/awf/ADBench
+## Graph Archive
 
-$env:path="$env:path;C:\Users\toelli\AppData\Local\Programs\Python\Python37"
-$env:path="$env:path;C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin"
-$env:build_sourcesdirectory="C:\Users\toelli\ADBench"
-$env:path="$env:path;C:\Users\toelli\ninja-win"
-
-git submodule init
-git submodule update
-```
-
-## Usage
-
-1) Run `powershell ADBench/run-all.ps1` to run all of the tools and write the timings to `/tmp/`.
-2) Run `python ADBench/plot_graphs.py` to plot graphs of the resulting timings and write them to `/Documents/New Figures/`
-
-### CLI reference: run-all
-
-`powershell ADBench/run-all.ps1 buildtype nruns_f nruns_J time_limit tmpdir repeat`
-- `buildtype`: The build configuration of the tests to be run (`Debug` or `Release`). By default, both will be run (pass an empty string `""` to replicate this).
-- `nruns_f`: Number of times to run the original functions (BA, GMM) for each tool (default = `10`)
-- `nruns_J`: Number of times to run the autodiff process for each tool (default = `10`)
-- `time_limit`: The maximum amount of time (in seconds) to spend benchmarking each tool (default = `60`)
-	- After each run, the program will check if it has exceeded the time limit
-	- `time_limit` will never cause a task to run less than once, and the program will always output a resultant time
-	- **Note**: The time limits are only implemented in C++ and Python currently
-- `timeout`: The maximum amount of time (in seconds) to allow each tool to run for
-	- If a program exceeds this time limit, PowerShell kills it
-	- This may result in no runs being completed, and no output being produced (PowerShell will output a file with `inf` timings in it to mark the test as failed)
-- `tmpdir`: The output directory to use (default = `/tmp/`)
-- `repeat`: Whether to repeat tasks for which an output file already exists (default = `false`)
-
-### CLI reference: plot_graphs
-
-`python plot_graphs.py [--save] [--plotly] [--show]`
-- `--save`: Save graphs as .png files to `Documents/New Figures/`
-- `--plotly`: Save graphs as [Plot.ly](https://plot.ly/) .html files to `Documents/New Figures/plotly/`
-- `--show`: Display graphs in new windows
-- If neither `--save` or `--plotly` are included, `--show` will be `True` by default - otherwise, it must be manually enabled
-
-### check_J
-
-`python check_J.py`
-- This script compares all available Jacobian output files, and outputs results to the console
-- It will flag missing output files, and indicate where there may be a mismatch between files
-- For comparison, all derivatives are rounded to a certain number of significant figures, as specified by a constant at the top of the script
+From time to time we run the benchmarks and publish the resulting plots here:
+https://adbenchwebviewer.azurewebsites.net/
```

Comment:
I think, it would be nice to put here a reference to the file `docs/AzureBatch.md`, described in the PR #149

Reply:
OK